verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -389,9 +397,13 @@ class LinelistTable extends React.Component<Props, LinelistTableState> {
                                                 geoResolution:
                                                     c.location?.geoResolution,
                                                 locationName: c.location?.name,
-                                                confirmedDate: confirmedDate
-                                                    ? new Date(confirmedDate)
+                                                confirmedDate: confirmedEvent
+                                                    ? new Date(
+                                                          confirmedEvent.dateRange.start,
+                                                      )
                                                     : null,
+                                                confirmationMethod:
+                                                    confirmedEvent?.value || '',
```

Comment:
Is the `|| ''` needed?

Reply:
ah no sorry that was before I used confirmedDate?: string as a type